src/users/schema/music.schema.ts

```diff
@@ -120,6 +121,15 @@ export class Music {
   })
   url: string;
 
+  /**
+   * Object Id of user who uploaded the music
+   * @example 5ff4c9d8e4b0f8b8b8b8b8b8
+   * @required
+   */
+  @ApiProperty({ example: '5ff4c9d8e4b0f8b8b8b8b8b8' })
+  @Prop({ type: SchemaTypes.ObjectId, required: true, auto: false })
+  userId: Types.ObjectId;
+
```

Comment:
At the moment, I used `ownerId` in schema and services too. But I might change it to `userId` later. So please change to `ownerId` when solving conflicts.

Reply:
Please make sure to change it into `userId` later, note that in playLisy schema we also named it `userId`.